Makefile

```diff
@@ -63,7 +63,7 @@ fuzz:
 # Static analysis
 
 verify: verify-gofmt verify-bom verify-lint verify-dep verify-shellcheck verify-goword \
-	verify-govet verify-license-header verify-receiver-name verify-mod-tidy verify-shellcheck \
+	verify-govet verify-license-header verify-receiver-name verify-mod-tidy \
```

Comment:
Why remove `verify-shellcheck`?

Reply:
This was a duplicate, refer to line 65 :)